lib/v2/chsi/maintainer.js

```diff
@@ -1,3 +1,4 @@
 module.exports = {
     '/kydt': ['SunBK201'],
+    '/hotnews': ['yanbot-team'],
```

Comment:
```suggestion
    '/hotnews': ['yanbot-team'],
    '/kydt': ['SunBK201'],
```
Insert new entries in alphabetic order.